core/src/torii/mod.rs

```diff
@@ -45,10 +44,10 @@ pub struct Torii<W: WorldTrait> {
 #[derive(Error, Debug)]
 pub enum Error {
     /// Failed to execute or validate query
-    #[error("Failed to execute or validate query")]
+    #[error("Failed to execute or validate query. {0}")]
```

Comment:
We shouldnt duplicate error output if we already have its source. It is printed here:
https://github.com/hyperledger/iroha/blob/iroha2-dev/core/src/torii/mod.rs#L98-L113

Reply:
Then it seems that the PR is not needed?

Reply:
I'm closing this PR, and #1502 until there's an MWE of a missing error body.